DFKPS/iocBoot/iocDFKPS-IOC-02/st-8000.cmd

```diff
@@ -0,0 +1,19 @@
+## set positions of interlock and power flags in status string
```

Comment:
Rather than replicate these files the files in IOC-01 can be called, which means any changes only need to be done once